docs/cookbook/html-conversion.md

```diff
@@ -74,7 +74,7 @@ npm install bootstrap
 This template uses Bootstrap 5, which is the latest version as of this writing. If you need another version for your template, make sure to specify it during the install.
 :::
 
-The next thing we will do is copy the contents of the `scss` folder into our project. There are multiple places that we could add these files, but the best place is in a project-level `modules/@postrophecms/asset` folder. If the folder doesn't already exist in your project, create it. Next, within that folder create a `ui/src` folder and copy the entirety of the `dist/scss` folder.
+The next thing we will do is copy the contents of the `scss` folder into our project. There are multiple places that we could add these files, but the best place is in a project-level `modules/asset` folder. If the folder doesn't already exist in your project, create it. Next, within that folder create a `ui/src` folder and copy the entirety of the `dist/scss` folder.
```

Comment:
This appears to contradict the code below which imports that scss from the npm package.

Reply:
We do want a process that imports from npm gracefully if possible rather than copying a bunch of files.

Reply:
I'll have to update the surrounding text to clarify. All of the bootstrap files are being imported from npm. These are theme-specific files that provide custom, non-bootstrap styling.